docs/Overview.th-TH.md

```diff
@@ -0,0 +1,124 @@
+---
+id: overview-th-TH
+title: Overview
+layout: docs
+category: Quick Start
+permalink: docs/overview-th-TH.html
+next: todo-list-th-TH
+lang: th-TH
+---
+
+Flux คือการออกแบบโครงสร้างแอพพลิเคชั่นที่ Facebook ใช้สร้าง web applications ที่ฝั่ง client โดยใช้ประโยชน์จากการที่ข้อมูลเดินทางในทิศทางเดียวซึ่งเหมาะกับ view components ของ React ที่สามารถประกอบกันได้ โดยที่ Flux เป็นเหมือนกับรูปแบบการทำงานมากกว่าจะเรียกว่าเป็น framework ซึ่งคุณสามารถเริ่มต้นใช้ Flux ได้เลยโดยไม่ต้องเขียนโค้ดเพิ่มมากนัก
```

Comment:
Another point to verify: I want to make sure this expresses that Flux is not a framework.  Flux should be considered to be a pattern instead.